src/App/Components/Separator/or_separator.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class Separator extends Component {
+  render() {
+    return (
+      <div className="or-separator">
+        <b>or</b>
```

Comment:
@MCFrank16 it would be nice if this component can receive props in case someone else wants to reuse it, with a different separator other than 'or'